### DISCUSSION TITLE
Brew Doctor - "config" scripts exist outside your system or Homebrew directories" + "Unbrewed header files" Error Messages

### DISCUSSION THREAD

**Discussion by laurencereeds** [Homebrew/discussions#536] [Getting started] [answered]
Hi,

I had to update Homebrew after I downloaded macOs Big Sur. I ran brew doctor to check if everything was ok and I ran into 2 different error messages.

1 - "config" scripts exist outside your system or Homebrew directories - **Issue related to PostGres**

" Having additional scripts in your path can confuse software installed via Homebrew if the config script overrides a system or Homebrew-provided script of the same name. We found the following "config" scripts: **/Applications/Postgres.app/Contents/Versions/latest/bin/gdal-config** "

2 - Unbrewed header files - **Issue related to node**

" Unbrewed header files were found in /usr/local/include. If you didn't put them there on purpose they could cause problems when building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/node/js_native_api.h
  /usr/local/include/node/js_native_api_types.h
  /usr/local/include/node/libplatform/libplatform-export.h
  /usr/local/include/node/libplatform/libplatform.h
  /usr/local/include/node/libplatform/v8-tracing.h
[...] "

Questions

1. Is it recommended that I fix the issues? There is a disclaimer right before the report "Please note that these warnings are just used to help the Homebrew maintainers with debugging if you file an issue. If everything you use Homebrew for is working fine: please don't worry or file an issue; just ignore this. Thanks! " The warning messages though seem important so I am little confused on what I should do.
2. If so, what should be the steps for correcting both issues?

Thank you in advance!

**Comment by gdolsson** [comment, marked as answer]
From other answers regarding brew doctor output I've seen so far, nothing. 

Unless you have a problem using/installing brew packages you should ignore these warnings.

I have yet found any problem with config files on my system and after working to resolve some issues, most other warnings seem to refer to files having been installed in "non-standard" locations or when files from programs installed outside of brew places files in usr/local/something

Though these are just best guesses based on experience from usage.

Best regards
// Gustaf

**Reply by SMillerDev** [reply]
The first line in `brew doctor` is as follows for this exact reason:
> Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!
